easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.2.2-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Xerces-C++'
+version = '3.2.2'
+
+homepage = 'http://xerces.apache.org/xerces-c/'
```

Comment:
The tests failed because the URLs can use `https`

Reply:
Changed to https